internal/core/converter/tensorflow/tensorflow.go

```diff
@@ -69,7 +70,17 @@ const (
 	threeDim
 )
 
-var loadFunc = tf.LoadSavedModel
+var loadFunc = func(t *tensorflow) error {
```

Comment:
local func variable is not clean, please use receiver func instead of local func val

Reply:
I think he want to override it during testing, so he defines this function as variable instead of the receiver function.

One more thing, this function is not tested ... (the coverage of this file is 86%)

Reply:
exactly, that's why I recommend method & it can be replaced by Interface, please mock it using interface